components/app.js

```diff
@@ -10,6 +11,12 @@ class App {
   }
   handleGetGradesSuccess(grades){
     this.gradeTable.updateGrades(grades);
+    var tempGrade = 0;
+    for(var i =0; i<grades.length;i++){
+      tempGrade += grades[i][`grade`];
```

Comment:
grades[i].grade should work any reason you indexed the object with a string?

Reply:
I tried tempGrade += grades[i][grades] at first but it didnt work.
so I tried this one and it worked tempGrade += grades[i][`grade`]